<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyConditionDialog</class>
 <widget class="QDialog" name="ModifyConditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification du noeud condition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="1">
    <widget class="QLabel" name="labelID">
     <property name="text">
      <string>IDDUSTEP</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="plainTextEditComment">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Commentaire  (affiché sous forme d'infobulle au survol du composant)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Couleur</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonColor</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hauteur</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxHeight</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Lien vers le détail</string>
     </property>
     <property name="buddy">
      <cstring>pushButtonLinkInit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Id. du noeud</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Largeur</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxWidth</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Infobulle</string>
     </property>
     <property name="buddy">
      <cstring>plainTextEditComment</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPlainTextEdit" name="plainTextEditCondition">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Texte de la condition (affiché dans le composant)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
     <item>
      <widget class="QPushButton" name="pushButtonLinkInit">
       <property name="text">
        <string>Parcourir...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLink">
       <property name="text">
        <string>Réinitialiser</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="QPushButton" name="pushButtonColor">
       <property name="text">
        <string>Modifier la couleur...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonColorInit">
       <property name="text">
        <string>Réinitialiser</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBoxWidth">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBoxHeight">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Condition</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>plainTextEditCondition</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>plainTextEditCondition</tabstop>
  <tabstop>plainTextEditComment</tabstop>
  <tabstop>spinBoxWidth</tabstop>
  <tabstop>spinBoxHeight</tabstop>
  <tabstop>pushButtonColor</tabstop>
  <tabstop>pushButtonColorInit</tabstop>
  <tabstop>pushButtonLinkInit</tabstop>
  <tabstop>pushButtonLink</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifyConditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifyConditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
